share/OpenMS/examples/FRACTIONS/BSA_design_onetable_nonconsec.tsv

```diff
@@ -0,0 +1,7 @@
+Fraction_Group	Fraction	Spectra_Filepath	Label	Sample	MSstats_Condition	MSstats_BioReplicate
+1	2	BSA1_F1.mzML	1	5	1	1
```

Comment:
You mentioned strings for some entries. Does this already work? If so can you provide a test that indicates where strings are supported (cols)